core/components/organisms/verticalNav/MenuItem.tsx

```diff
@@ -98,11 +92,12 @@ export const MenuItem = (props: MenuItemProps) => {
     ['MenuItem--disabled']: menu.disabled,
     ['MenuItem--subMenu']: isChildren && expanded,
     ['MenuItem--rounded']: rounded && expanded,
+    [`color-${itemColor}`]: true,
```

Comment:
Please check if we can add test cases for this class.

Reply:
I have added test case for this class already.